spec/classes/chrony_spec.rb

```diff
@@ -40,6 +40,22 @@
             it { is_expected.to contain_file('/etc/chrony.keys').with_replace(true) }
             it { is_expected.to contain_file('/etc/chrony.keys').with_content("0 xyzzy\n") }
           end
+        when 'Gentoo'
+          context 'using defaults' do
+            it { is_expected.to contain_file('/etc/chrony/chrony.conf').without_content(%r{^\s*cmdallow}) }
+            ['0.pool.ntp.org', '1.pool.ntp.org', '2.pool.ntp.org', '3.pool.ntp.org'].each do |s|
+              it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*server #{s} iburst$}) }
+            end
+            it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*rtconutc$}) }
+            it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*driftfile /var/lib/chrony/drift$}) }
+            it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*rtcsync$}) }
+            it { is_expected.to contain_file('/etc/chrony/chrony.conf').without_content(%r{^\s*dumpdir}) }
+            it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_mode('0644') }
+            it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_owner('0') }
+            it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_group('0') }
+            it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_replace(true) }
+            it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_content("0 xyzzy\n") }
```

Comment:
Same here. Multiple `it` statements require the catalog to be compiled multiple times and is much slower in my experience.

Reply:
Same reply here, it should be fixed for all osfamily.

Reply:
my suggestion:
fix for Gentoo as suggested by @ekohl, fix for other OSes in new pr
add a TODO comment, if you're the forgetful type ;)

Reply:
Ok, I'll do that.